test/test.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+const {TestRunner, Reporter, Matchers} = require('@pptr/testrunner');
+const Tracium = require('..');
+
+const readFileAsync = util.promisify(fs.readFile);
+const readJSON = async filePath => {
+  const text = await readFileAsync(path.join(__dirname, filePath), 'utf8');
+  return JSON.parse(text);
+}
+
+const testRunner = new TestRunner();
+const {describe, xdescribe, fdescribe} = testRunner;
+const {it, fit, xit} = testRunner;
+const {beforeAll, beforeEach, afterAll, afterEach} = testRunner;
+const {expect} = new Matchers();
+
+describe('Tracium tests', function() {
+  it('should work with no FCP', async () => {
+    const trace = await readJSON('no_fcp.trace.json');
+    expect(Tracium.computeMainThreadTasks(trace).length).toBeGreaterThan(1);
```

Comment:
⚠️ long tangent ahead feel free to reply only if you have time :)

we have fights about this over in LH so I'm curious what your take is...

some people like to do something like `expect(Tracium.computeMainThreadTasks(trace).length).toMatchInlineSnapshot(452)`

others think it absurdly stretches the bounds of automatically updatable expectations

(`toMatchInlineSnapshot` is a jest automatically updatable inline golden file essentially if you're unfamiliar with the command)

between what you have here, `toEqual(<manually computed number>)`, and `toMatchInlineSnapshot(<autocomputed number>)` what wins in the mind of @aslushnikov and why :)

Reply:
@patrickhulce (I started writing an answer yesterday night but fall asleep)

> between what you have here, toEqual(<manually computed number>), and toMatchInlineSnapshot(<autocomputed number>) what wins in the mind of @aslushnikov and why :)

The `toEqual(<manual number>)` and `toMatchInlineSnapshot(<autocomputed number>)` are basically the same thing - once the test fails, an engineer will have to look into Tracium, than look into the test, and figure who's fault is this. 

Given how many tasks we have here (452?), debugging this test failure might be a tedious process! If we have a dozen of such tests, each asserting some exact number of tasks, than the **effort of maintaining and acting** on the test failures will accumulate quickly.

The decision to **accept** or to **not accept** this effort is defined by the level of investment into the project and the risks we want to mitigate. If we were to build a core module for rocket launching, we'd build and exhaustive test coverage to catch as much as possible. We'd be fine with all the noise we have from the tests - better safe than sorry! 

In real world, most project don't deal with that much risks. If tests are unreasonably strict, developers will get frustrated and will just rubber-stamp-pass them. These tests suck out all the fun and have no value!

So my rule of a thumb is to start small and build up test coverage according to the needs. This particular test is added to avoid regressions once Tracium is merged into LH - and it seems to be doing just enough for this goal :-)